ERROR - 2025-07-22 07:54:10 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-07-22 07:54:10 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:54:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kabupaten`
WHERE `kabupaten`.`id_kabupaten` = :kabupaten.id_kabupaten:', [...], false)
 3 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
 5 APPPATH\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PetaController->publik()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-22 07:54:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kabupaten`
WHERE `kabupaten`.`id_kabupaten` = :kabupaten.id_kabupaten:', [...], false)
 3 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
 5 APPPATH\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PetaController->publik()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:54:20 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\Dashboard.php(41): CodeIgniter\BaseModel->find(1)
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\Dashboard->aoc()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\Dashboard.php(41): CodeIgniter\BaseModel->find(1)
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\Dashboard->aoc()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:54:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: dashboard_aoc]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kabupaten`
WHERE `kabupaten`.`id_kabupaten` = :kabupaten.id_kabupaten:', [...], false)
 3 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
 5 APPPATH\Controllers\Dashboard.php(41): CodeIgniter\BaseModel->find(1)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->aoc()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:54:20 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\Dashboard.php(41): CodeIgniter\BaseModel->find(1)
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\Dashboard->aoc()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\Dashboard.php(41): CodeIgniter\BaseModel->find(1)
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\Dashboard->aoc()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:54:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: dashboard_aoc]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kabupaten`
WHERE `kabupaten`.`id_kabupaten` = :kabupaten.id_kabupaten:', [...], false)
 3 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
 5 APPPATH\Controllers\Dashboard.php(41): CodeIgniter\BaseModel->find(1)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->aoc()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:54:34 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:54:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kabupaten`
WHERE `kabupaten`.`id_kabupaten` = :kabupaten.id_kabupaten:', [...], false)
 3 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
 5 APPPATH\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PetaController->publik()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:54:34 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:54:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kabupaten`
WHERE `kabupaten`.`id_kabupaten` = :kabupaten.id_kabupaten:', [...], false)
 3 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
 5 APPPATH\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PetaController->publik()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:54:40 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:54:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:54:42 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:54:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:55:32 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:55:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 07:56:08 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 07:56:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 08:04:49 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 08:04:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 08:13:09 --> mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-07-22 08:13:09 --> CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-22 08:13:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user`
WHERE `username` = \'kukuhj\'
AND `is_active` = 1
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user`
WHERE `username` = \'kukuhj\'
AND `is_active` = 1
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-22 08:13:09 --> [Caused by] mysqli_sql_exception: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `user`
WHERE `username` = \'kukuhj\'
AND `is_active` = 1
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `user`
WHERE `username` = \'kukuhj\'
AND `is_active` = 1
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `user`
WHERE `username` = \'kukuhj\'
AND `is_active` = 1
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
AND `is_active` = :is_active:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\AuthController.php(20): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->doLogin()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 08:13:26 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'aoc', 3306, '', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#6 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tawannarkobav2\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
#5 C:\xampp\htdocs\tawannarkobav2\app\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
#6 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\PetaController->publik()
#7 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-07-22 08:13:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kabupaten`
WHERE `kabupaten`.`id_kabupaten` = :kabupaten.id_kabupaten:', [...], false)
 3 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, 1)
 5 APPPATH\Controllers\PetaController.php(28): CodeIgniter\BaseModel->find(1)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PetaController->publik()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PetaController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-22 08:26:30 --> mysqli_sql_exception: Duplicate entry 'U11' for key 'PRIMARY' in C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\tawannarkobav2\system\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE `ungkap_...', 0)
#1 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `ungkap_...')
#2 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `ungkap_...')
#3 C:\xampp\htdocs\tawannarkobav2\system\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `ungkap_...', Array, false)
#4 C:\xampp\htdocs\tawannarkobav2\system\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\tawannarkobav2\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\tawannarkobav2\system\Model.php(854): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\tawannarkobav2\app\Controllers\Ungkapkasus.php(134): CodeIgniter\Model->update('U100', Array)
#8 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(933): App\Controllers\Ungkapkasus->update('U100')
#9 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ungkapkasus))
#10 C:\xampp\htdocs\tawannarkobav2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tawannarkobav2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tawannarkobav2\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-22 08:26:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'U11' for key 'PRIMARY'
[Method: POST, Route: ungkapkasus/update/U100]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `ungkap_kasus` SET `id_kasus` = :id_kasus:, `Nama` = :Nama:, `JK` = :JK:, `Umur` = :Umur:, `golonganumur` = :golonganumur:, `Pekerjaan` = :Pekerjaan:, `BB` = :BB:, `JmlBB` = :JmlBB:, `Satuan` = :Satuan:, `MO` = :MO:, `Tahun` = :Tahun:, `id_kelurahan` = :id_kelurahan:, `Jam` = :Jam:, `TKP` = :TKP:
WHERE `ungkap_kasus`.`id_kasus` IN :ungkap_kasus.id_kasus:', [...], false)
 2 SYSTEMPATH\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Ungkapkasus.php(134): CodeIgniter\Model->update('U100', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ungkapkasus->update('U100')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ungkapkasus))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-22 08:26:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'U11' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `ungkap_kasus` SET `id_kasus` = \'U11\', `Nama` = \'Eko Yanuar Saputra\', `JK` = \'L\', `Umur` = \'24\', `golonganumur` = \'Dewasa\', `Pekerjaan` = \'Wiraswasta\', `BB` = \'Tembakau Sintetis\', `JmlBB` = \' 65,0 \', `Satuan` = \'Gram\', `MO` = \'Penjual\', `Tahun` = \'2022\', `id_kelurahan` = \'kl139\', `Jam` = \'00:00:23\', `TKP` = \'Jl Kelud Dekat Simpang Empat Karangsalam\'
WHERE `ungkap_kasus`.`id_kasus` IN (\'U100\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `ungkap_kasus` SET `id_kasus` = \'U11\', `Nama` = \'Eko Yanuar Saputra\', `JK` = \'L\', `Umur` = \'24\', `golonganumur` = \'Dewasa\', `Pekerjaan` = \'Wiraswasta\', `BB` = \'Tembakau Sintetis\', `JmlBB` = \' 65,0 \', `Satuan` = \'Gram\', `MO` = \'Penjual\', `Tahun` = \'2022\', `id_kelurahan` = \'kl139\', `Jam` = \'00:00:23\', `TKP` = \'Jl Kelud Dekat Simpang Empat Karangsalam\'
WHERE `ungkap_kasus`.`id_kasus` IN (\'U100\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `ungkap_kasus` SET `id_kasus` = :id_kasus:, `Nama` = :Nama:, `JK` = :JK:, `Umur` = :Umur:, `golonganumur` = :golonganumur:, `Pekerjaan` = :Pekerjaan:, `BB` = :BB:, `JmlBB` = :JmlBB:, `Satuan` = :Satuan:, `MO` = :MO:, `Tahun` = :Tahun:, `id_kelurahan` = :id_kelurahan:, `Jam` = :Jam:, `TKP` = :TKP:
WHERE `ungkap_kasus`.`id_kasus` IN :ungkap_kasus.id_kasus:', [...], false)
 4 SYSTEMPATH\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Ungkapkasus.php(134): CodeIgniter\Model->update('U100', [...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ungkapkasus->update('U100')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ungkapkasus))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-22 08:26:30 --> [Caused by] mysqli_sql_exception: Duplicate entry 'U11' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE `ungkap_kasus` SET `id_kasus` = \'U11\', `Nama` = \'Eko Yanuar Saputra\', `JK` = \'L\', `Umur` = \'24\', `golonganumur` = \'Dewasa\', `Pekerjaan` = \'Wiraswasta\', `BB` = \'Tembakau Sintetis\', `JmlBB` = \' 65,0 \', `Satuan` = \'Gram\', `MO` = \'Penjual\', `Tahun` = \'2022\', `id_kelurahan` = \'kl139\', `Jam` = \'00:00:23\', `TKP` = \'Jl Kelud Dekat Simpang Empat Karangsalam\'
WHERE `ungkap_kasus`.`id_kasus` IN (\'U100\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `ungkap_kasus` SET `id_kasus` = \'U11\', `Nama` = \'Eko Yanuar Saputra\', `JK` = \'L\', `Umur` = \'24\', `golonganumur` = \'Dewasa\', `Pekerjaan` = \'Wiraswasta\', `BB` = \'Tembakau Sintetis\', `JmlBB` = \' 65,0 \', `Satuan` = \'Gram\', `MO` = \'Penjual\', `Tahun` = \'2022\', `id_kelurahan` = \'kl139\', `Jam` = \'00:00:23\', `TKP` = \'Jl Kelud Dekat Simpang Empat Karangsalam\'
WHERE `ungkap_kasus`.`id_kasus` IN (\'U100\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `ungkap_kasus` SET `id_kasus` = \'U11\', `Nama` = \'Eko Yanuar Saputra\', `JK` = \'L\', `Umur` = \'24\', `golonganumur` = \'Dewasa\', `Pekerjaan` = \'Wiraswasta\', `BB` = \'Tembakau Sintetis\', `JmlBB` = \' 65,0 \', `Satuan` = \'Gram\', `MO` = \'Penjual\', `Tahun` = \'2022\', `id_kelurahan` = \'kl139\', `Jam` = \'00:00:23\', `TKP` = \'Jl Kelud Dekat Simpang Empat Karangsalam\'
WHERE `ungkap_kasus`.`id_kasus` IN (\'U100\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `ungkap_kasus` SET `id_kasus` = :id_kasus:, `Nama` = :Nama:, `JK` = :JK:, `Umur` = :Umur:, `golonganumur` = :golonganumur:, `Pekerjaan` = :Pekerjaan:, `BB` = :BB:, `JmlBB` = :JmlBB:, `Satuan` = :Satuan:, `MO` = :MO:, `Tahun` = :Tahun:, `id_kelurahan` = :id_kelurahan:, `Jam` = :Jam:, `TKP` = :TKP:
WHERE `ungkap_kasus`.`id_kasus` IN :ungkap_kasus.id_kasus:', [...], false)
 5 SYSTEMPATH\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Ungkapkasus.php(134): CodeIgniter\Model->update('U100', [...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ungkapkasus->update('U100')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ungkapkasus))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(58): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
